config/cesm/machines/config_workflow.xml

```diff
@@ -54,11 +54,11 @@
     <job name="timeseries">
       <template>$ENV{POSTPROCESS_PATH}/timeseries/template.timeseries</template>
       <dependency>case.st_archive</dependency>
-      <prereq>$TIMESERIES</prereq>
+      <prereq>1</prereq>
```

Comment:
Is this the correct prereq.  I just want to make sure you didn't set it to 1 for debugging.

Reply:
It's hardcoded but it won't affect anyone not using --workflow timeseries